tests/test_perf_tracking.py

```diff
@@ -2085,3 +2085,62 @@ def test_update_positions(self):
         # Test gross and net exposures
         self.assertEqual(100 + 150000 + 200, pos_stats.gross_exposure)
         self.assertEqual(100 + 150000 - 200, pos_stats.net_exposure)
+
+    def test_close_position(self):
+        future_sid = 1032201401
+        equity_sid = 1
+        pt = perf.PositionTracker(self.env.asset_finder, None)
+        dt = pd.Timestamp('2017/01/04 3:00PM')
+
+        pos1 = perf.Position(
+            sid=future_sid,
+            amount=np.float64(30.0),
+            last_sale_date=dt,
+            last_sale_price=100,
+        )
+        pos2 = perf.Position(
+            sid=equity_sid,
+            amount=np.float64(10.0),
+            last_sale_date=dt,
+            last_sale_price=10,
+        )
+
+        # Update the positions dictionary with `future_sid` first. The order
+        # matters because it affects the multipliers dictionaries, which are
+        # OrderedDicts. If `future_sid` is not removed from the multipliers
+        # dictionaries, equities will hit the incorrect multiplier when
+        # computing `pt.stats()`.
+        pt.update_positions({future_sid: pos1, equity_sid: pos2})
+
+        for sid in (future_sid, equity_sid):
+            self.assertIn(sid, pt._position_value_multipliers)
```

Comment:
I'm 50/50 on this check.

Generally, it's best to avoid accessing semantically "private" members in tests, since those members are implementation details and could change in the future.

However, checking the value does get at the root cause of the bug.

Checking the `pt.stats()` values as you do below may be enough.